package/scripts/install_prereqs_for_wheels.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+install_prereqs_for_mac() {
```

Comment:
Replace the install_prerews for Mac from here - https://github.com/GenomicsDB/GenomicsDB-Python/blob/aa1af3ffacc99e9a97e36b68b953c9c316252f29/package/publish_package_local.sh#L55